src/include/daos/dtx.h

```diff
@@ -39,10 +39,14 @@ struct daos_tx_id {
 };
 
 static inline void
-daos_generate_dti(struct daos_tx_id *dti)
+daos_generate_dti(struct daos_tx_id *dti, bool zero)
 {
-	uuid_generate(dti->dti_uuid);
-	dti->dti_sec = time(NULL);
+	if (zero) {
+		memset(dti, 0, sizeof(*dti));
+	} else {
+		uuid_generate(dti->dti_uuid);
+		dti->dti_sec = time(NULL);
```

Comment:
Please add comment and say we should switch to HLC at here.

Reply:
OK